app/web/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -186,17 +192,7 @@ export default function GroupChatView() {
             </HeaderButton>
 
             <PageTitle className={classes.title}>
-              {groupChat ? (
-                groupChatTitleText(
-                  groupChat,
-                  groupChatMembersQuery,
-                  currentUserId
-                )
-              ) : groupChatError ? (
-                "Error"
-              ) : (
```

Comment:
We should probably still show an error alert when `groupChatError` is not null (with whatever in `groupChatError.message` rather than this state being completely ignored/removed in the new code.

Reply:
I added generic error message to alert when groupCharError is not null but groupChatError.message is not provided. The error message should not be in the page title imho

Reply:
Ahh I didn't realise `groupChatError.message` was already being displayed in the error alert as mentioned. Good to have a default message fallback too rather than the empty string before 👏